Update build.sh

startergo opened the pull request:
Fix previous.

startergo committed:
Update build.sh

Fix previous.

xaionaro commented:
Do I understand correctly you removed the comment line and added a space to other 3 lines?

startergo commented:
> Do I understand correctly you removed the comment line and added a space to other 3 lines?

I removed the first part along with the comment line ( I will reinstate the comment line):
https://github.com/xaionaro/edk2-builder-docker/pull/31/files#diff-26aa13992544ed8e184d72cdd63824affa0109cb1e44298b44b2495b4899e789L9-L12.
Also during building of debug version I get this error:
```

/home/edk2/edk2/RefindPlusPkg/Library/MemLogLib/BootLog.c:384:13: error: function declared 'ms_abi' here was previously declared without calling convention
VOID EFIAPI DebugLog (
            ^
/home/edk2/edk2/RefindPlusPkg/BootMaster/global.h:574:6: note: previous declaration is here
VOID DebugLog (
     ^
1 error generated.
make: *** [GNUmakefile:5138: /home/edk2/Build/RefindPlus/DEBUG_CLANG38/X64/RefindPlusPkg/RefindPlus/OUTPUT/Library/MemLogLib/BootLog.obj] Error 1


build.py...
 : error 7000: Failed to execute command
    make tbuild [/home/edk2/Build/RefindPlus/DEBUG_CLANG38/X64/RefindPlusPkg/RefindPlus]


build.py...
 : error F002: Failed to build module
    /home/edk2/edk2/RefindPlusPkg/RefindPlus.inf [X64, CLANG38, DEBUG]

- Failed -
Build end time: 19:37:52, Dec.27 2022
Build total time: 00:00:23
```

It looks like changing ```VOID DebugLog``` to ```extern VOID EFIAPI DebugLog``` in global.h should patch for now until it is fixed in the next release.
